nss-day-cohort-41/tabloidmvc-the-gefilte-fish
# Delete comment

# Description
- When user visits the comments page, they should see the delete option on their comments
- When 'Delete' button is clicked, user is directed to the delete confirmation page
- The delete confirmation page displays the Subject and Content of the comment along with two buttons 'Delete' and 'Cancel'
- When 'Delete' is clicked the comment is removed form the database and the user is re-directed back to comments page
- When 'Cancel' is clicked the user is re-directed back to the comments page
- Ticket #21 
Fixes # (issue)
- Modified Index method in CommentController to invoke GetPostById method instead of GetPublishedPostById. This fixes issue of null object found for Model.Post.Title.
## Type of change
Please delete options that are not relevant.
# Testing Instructions
- Navigate to comments page on any Post
- Make sure current user has at least 1 comment
- Current user should see delete button
- Clicking delete will direct to delete confirmation page
- Click delete to remove comment or cancel to go back to comments page
# Checklist:
- [ ] My changes generate no new warnings or errors -- Edit button is not functioning yet and will throw exception
